packages/ts/form/src/Validators.ts

```diff
@@ -310,7 +310,7 @@ export class Digits<T> extends AbstractValidator<T> {
 
   override validate(value: T): boolean {
     return (
-      String(toFloat(String(value))).indexOf('.') <= this.integer &&
+      String(Math.abs(toFloat(String(value)))).indexOf('.') <= this.integer &&
```

Comment:
`.indexOf()` returns `-1` if there is no decimal separator. Should we do this instead?
```suggestion
      String(Math.floor(Math.abs(toFloat(String(value))))).length <= this.integer &&
```